scripts/change-log-generator.js

```diff
@@ -106,23 +80,38 @@ function validateReleaseBranch(releaseBranch) {
   }
 }
 
+/**
+ * Create the changelog branch for committing these changes.
+ * If the branch already exists, check it out to append any
+ * new changes.
+ */
 function getNewChangeLogBranch(releaseBranch) {
   var changeLogBranch =
     CHANGE_LOG_BRANCH + releaseBranch.replace('origin/release/v', '');
-  shell.exec(util.format(GIT_CHECKOUT, changeLogBranch, releaseBranch));
+  shell.exec(
+    util.format(
+      "git checkout $(git show-ref --verify --quiet refs/heads/%s || echo '-b') %s",
```

Comment:
Runs 'git checkout -b' if the branch does not currently exist. Otherwise, checks out the existing branch.